addons/zylann.hterrain/tools/plugin.gd

```diff
@@ -852,6 +883,59 @@ func _debug_spawn_collider_indicators():
 				mi.position = hit.position
 
 
+func _show_brush_editor_overlay(min_size, max_size, widget_color, initial_value, action_name,
+									on_value_selected):
+	if _widget_action_name != action_name:
+		var active_viewport = _get_editor_viewport()
+
+		if active_viewport == null:
+			# when mouse outside 3d viewport area, dont try to show widget
+			return
+
+		_widget_action_name = action_name
+		var selector = HT_BrushEditorOverlayScene.instantiate()
+
+		selector.min_value = min_size
+		selector.max_value = max_size
+		selector.call_deferred("set_overlay_name", action_name)
+		selector.call_deferred("set_brush_preview_color", widget_color)
```

Comment:
You could make those not deferred by calling them after adding the selector to the tree.

Reply:
Didn't know that would work. I thought the @onready/ready was only called at the end of the frame.
Will change that if it works.